javascript/utils.js

```diff
@@ -0,0 +1,79 @@
+import {REGEX_EMAIL, REGEX_PASSWORD} from './regex.js';
+import {inputBoxEmail, inputPassword, eyeImg} from './constants.js';
+
+
+async function checkUsableEmail(givenEmail) {
+    try {
+        const signupEmail = {
+            email: givenEmail,
+        };
+        const response = await fetch('https://bootcamp-api.codeit.kr/api/check-email', {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(signupEmail),
+        })
+        if(response.ok) {
+            return true;
```

Comment:
c: 동작은 하지만 불필요한 true 반환으로 보이네요.

```suggestion
            return;
```